### PR TITLE
New cutText method

### PR DESCRIPTION
**Pull request by poogaboo**
Created new method: cutText
Updated class test to pass new method

### CODE REVIEW

File: force-app/main/default/classes/Utilities.cls [hunk 1]
```diff
@@ -13,4 +13,12 @@ public without sharing class Utilities {
         }
         return sobject_type_name;
     }
+
+    public static String cutText(String text, Integer numChar){
```
Comment (Raspikabek) [inline force-app/main/default/classes/Utilities.cls]: - Give a proper name to the new method
- method variables in snake_case
- it's necessary a converted variable? Can you reuse `text` variable?
- if statements like:
```
if (statement) { // whitespaces before and after parenthesis 
   //do something
}
```

File: force-app/main/default/classes/UtilitiesTest.cls [hunk 2]
```diff
@@ -14,4 +14,11 @@ private with sharing class UtilitiesTest {
         String sobject_name = Utilities.getSObjectNameById(test_account.Id);
         System.assertEquals('Account', sobject_name, 'Expecting Account SObject Type');
     }
+
+    @isTest
+    private static void testCutText(){
+        String myText = '1234567890';
```
Comment (Raspikabek) [inline force-app/main/default/classes/UtilitiesTest.cls]: - method variables in snake_case

File: force-app/main/default/classes/Utilities.cls (hunk 1 above)
Comment (poogaboo) [reply force-app/main/default/classes/Utilities.cls]: Qué tiquismiquis. El próximo me lo valida María X¬P

File: force-app/main/default/classes/Utilities.cls [hunk 3]
```diff
@@ -13,4 +13,8 @@ public without sharing class Utilities {
         }
         return sobject_type_name;
     }
+
+    public static String getCuttedString(String my_string, Integer max_length){
+        return (String.isBlank(my_string) || max_length == null || max_length >= my_string.length()) my_string : my_string.substring(0, max_length);
```
Comment (Raspikabek) [inline force-app/main/default/classes/Utilities.cls]:
```
return (String.isBlank(my_string) || max_length == null || max_length >= my_string.length()) ? my_string : my_string.substring(0, max_length);
```

**Comment by Raspikabek** [comment]
Esperando a arreglar travis.yml antes de aceptar nada adicional. Cuando este actualizado y subido a master avisare para que hagais merge the master y traer el fichero actualizado antes de aceptar nada.

**Comment by Raspikabek** [comment]
Travis config fixed and it's already updated in master (also in QA). 

Merge updates from this branches and I'll approve this pr

**Pull request merged**